.github/workflows/all_plugins.yaml

```diff
@@ -73,8 +71,16 @@ jobs:
     - name: "Set env keys"
       env: 
         STRIPE_PUBLISHABLE_KEY: ${{ secrets.STRIPE_PUBLISHABLE_KEY }}
+        STRIPE_SECRET_KEY: ${{ secrets.STRIPE_SECRET_KEY }}
       run: |
         ./.github/workflows/scripts/env-files.sh
 
+
+    - name: "Start dev server"
+      run: cd example/server/ && npm install && (npm run start:dev &) && cd ..
+
+    - name: "Set IP address"
+      run: echo "const kApiUrl='$(ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1')';" > integration_test/ip.dart
+
     - name: "Run test"
-      run: melos run test
```

Comment:
Leave 
```yaml
- name: "Run test"      
   run: melos run test;
```
for the unit tests before the integration tests (commented if you want for now)

Reply:
tried, doesn't run the correct flutter drive command out of the box

Reply:
Yeah, I mean separate it into two 
- name: "Run unit test"      
   run: melos run test;
- name: "Run integration test"      
   run: flutter run .....;

Check the melos.yaml file in the root level. I defined the test command myself. But for integration test we don't need melos as we will only test `flutter_stripe` no?